vm/src/syscalls.rs

```diff
@@ -16,8 +15,27 @@ pub struct Syscalls {
     log_buffer: Vec<Vec<u8>>,
     input: VecDeque<u8>,
     output: Vec<u8>,
+    label: Vec<Vec<u8>>,
 }
 
+pub enum SyscallCode {
```

Comment:
Would highly appreciate scoping PRs next time, as this change with documentation was not necessary.

Reply:
Sure will do.